src/current/v23.2/admission-control.md

```diff
@@ -79,12 +111,17 @@ Admission control works on the level of each node, not at the cluster level. The
 
 Organizing operations by priority can mean that higher priority operations consume all the available resources while lower priority operations remain in the queue until the operation times out.
 
+## Considerations
+
+Too many connections per [gateway node]({% link {{ page.version.version }}/architecture/sql-layer.md %}#gateway-node) can also lead to cluster overload. [Client connections]({% link {{ page.version.version }}/connection-parameters.md %}) are not managed by the admission control subsystem. {% include {{page.version.version}}/sql/server-side-connection-limit.md %}
```

Comment:
This may be a bit clearer

```suggestion
[Client connections]({% link {{ page.version.version }}/connection-parameters.md %}) are not managed by the admission control subsystem. Too many connections per [gateway node]({% link {{ page.version.version }}/architecture/sql-layer.md %}#gateway-node) can also lead to cluster overload. 

{% include {{page.version.version}}/sql/server-side-connection-limit.md %}
```

Reply:
love this edit - much clearer, thank you!